### DISCUSSION TITLE
Song Request - Same Artist failed

### DISCUSSION THREAD

**Discussion by AsaTyr2018** [AzuraCast/AzuraCast#7164] [Q&A] [answered]
Good day,

i use AzureCast to stream my own songs (Self Made songs with me as Artist) 
But the Song Request blocks every request due to the same artist.
![grafik](https://github.com/AzuraCast/AzuraCast/assets/43162495/ef9677cd-88bf-4a72-b78d-29380a285f69)
"Cannot submit request: This song or artist was played too recently. Please wait a while before requesting it again."

Any way to change this?

Kind regards
Asatyr

**Comment by gAlleb** [comment, marked as answer]
Profile -> Edit Profile -> AutoDJ and change to your taste.
<img width="500" alt="Снимок экрана 2024-06-02 в 08 30 19" src="https://github.com/AzuraCast/AzuraCast/assets/144447266/e4366df1-b1f8-4da1-adc9-44ff85f31a2f">
<img width="500" alt="Снимок экрана 2024-06-02 в 08 34 03" src="https://github.com/AzuraCast/AzuraCast/assets/144447266/3aef5eee-476a-45ce-84a7-d90944700c85">

**Reply by AsaTyr2018** [reply]
> Profile -> Edit Profile -> AutoDJ and change to your taste. <img alt="Снимок экрана 2024-06-02 в 08 30 19" width="500" src="https://private-user-images.githubusercontent.com/144447266/335842059-e4366df1-b1f8-4da1-adc9-44ff85f31a2f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTczMTgwNDAsIm5iZiI6MTcxNzMxNzc0MCwicGF0aCI6Ii8xNDQ0NDcyNjYvMzM1ODQyMDU5LWU0MzY2ZGYxLWIxZjgtNGRhMS1hZGM5LTQ0ZmY4NWYzMWEyZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYwMlQwODQyMjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xOGExMjI5N2UwYmVlNWY3MzkzOGRjZDIyYTQ4ZTU1ZGU1ZGIzZDkyYjg5NGE5NWI1MDRiMjMzMDY2YWFiNzEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.K1b6YFB3NuCy6HGaqwH5qb9EG37SyY0qhnVFx3TS2dU"> <img alt="Снимок экрана 2024-06-02 в 08 34 03" width="500" src="https://private-user-images.githubusercontent.com/144447266/335842181-3aef5eee-476a-45ce-84a7-d90944700c85.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTczMTgwNDAsIm5iZiI6MTcxNzMxNzc0MCwicGF0aCI6Ii8xNDQ0NDcyNjYvMzM1ODQyMTgxLTNhZWY1ZWVlLTQ3NmEtNDVjZS04NGE3LWQ5MDk0NDcwMGM4NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYwMlQwODQyMjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYjM0OTYyM2Y5NDg1MjBiZDc4ODdjYjA2ZTFlOTcyZmJjNTc1NGIxMGVmOWFkOTZhZTY0MDkzOTczZDA1ODRkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.EtNJUjFrsGDcG9g9uaEdhirX6RW0S9E2AKkVWzguYLU">

Oh god this was too simple xD 
I set the AutoDJ Settings like you said but missed in Song Requests.....

Thanks a lot! =D

**Comment by AsaTyr2018** [comment]
Working now. Thanks!
